underscore.js

```diff
@@ -647,6 +647,23 @@
     return range;
   };
 
+  // Split an **array** into several arrays containing **count** or less elements
+  // of initial array
+  _.chunk = function(array, count) {
+    count = count || 1;
```

Comment:
I think lodash has already set the prior art and breaking from it just causes compatibility headaches. We can discuss in [underdash](https://github.com/underdash/underdash/issues/4)?